src/components/representative.js

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+
+const Representative = props => {
+  // Question: do we want to add website or photo to this?
```

Comment:
Good question - I'm not sure, I guess we'll need to wait for more designs but I'll make sure Colleen know's we have images available. It's cool the API provides a photo URL and you implemented it here!